tests/v2/validation/nodescaling/scaling_nodepools.go

```diff
@@ -45,3 +48,39 @@ func ScalingRKE1NodePools(t *testing.T, client *rancher.Client, clusterID string
 	_, err = rke1.ScaleNodePoolNodes(client, cluster, node, nodeRoles)
 	require.NoError(t, err)
 }
+
+func ScalingAKSNodePools(t *testing.T, client *rancher.Client, clusterID string, nodePool *aks.NodePool) {
+	cluster, err := client.Management.Cluster.ByID(clusterID)
+	require.NoError(t, err)
+
+	clusterResp, err := aks.ScalingAKSNodePoolsNodes(client, cluster, nodePool)
+	require.NoError(t, err)
+
+	*nodePool.NodeCount = -*nodePool.NodeCount
```

Comment:
nit; now that all the work is basically in <hostedprovider>.ScalingNodePoolsNodes() I feel like the scaling back to the original value could directly be part of the test instead of in this helper, but could see this go either way.

Reply:
I personally think it's better to just have it as is, just because it's consistent with the Rancher clusters. That way, it's easier for users of the framework to read.